connector/protobuf/pom.xml

```diff
@@ -110,6 +109,47 @@
           </relocations>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.github.os72</groupId>
+        <artifactId>protoc-jar-maven-plugin</artifactId>
```

Comment:
@rangadi we might need to add protoc sbt plugin to generate test .proto class's, spark project uses this sbt-protoc plugin. https://github.com/thesamet/sbt-protoc and sbt file https://github.com/apache/spark/blob/master/project/SparkBuild.scala#L720

Reply:
Ack, thanks. My sbt build has been failing. Need to get that working.

Reply:
Updated sbt to generate java classes for tests.